app/views/general/_index.html.erb

```diff
@@ -3,16 +3,18 @@
   new_item_path = polymorphic_path([:new, controller_name.singularize.to_sym])
   items ||= instance_variable_get("@#{controller_name}")
   show_new_button = true unless local_assigns.has_key?(:show_new_button)
+  show_query_button = false unless local_assigns.has_key?(:show_query_button)
   title ||= nil
   subtitle ||= nil
 	isa_templates_enabled = Seek::Config.sample_type_template_enabled && Seek::Config.project_single_page_advanced_enabled
 %>
 
 <div class="pull-right">
+  <% if show_query_button && isa_templates_enabled %>
```

Comment:
it would clearer to use `isa_templates_enabled` to pass and set `show_query_button` flag
the flag should also have a clearer name, since this page is used for all types - e.g `show_samples_query_button`

Reply:
Agreed. Thanks for the comments.
This change request is in line with the plans to collect all ISA-JSON export-related features (including ST Templates and what is now called SP advanced) under a single configuration option, see discussions on issues #1602 and #1604 .